ebos/eclbasevanguard.hh

```diff
@@ -424,9 +445,22 @@ protected:
     void updateCartesianToCompressedMapping_()
     {
         size_t num_cells = asImp_().grid().leafGridView().size(0);
-        for (unsigned i = 0; i < num_cells; ++i) {
-            unsigned cartesianCellIdx = cartesianIndex(i);
-            cartesianToCompressed_[cartesianCellIdx] = i;
+        is_interior_.resize(num_cells);
```

Comment:
Just as an apropos/tangential remark, this `num_cells` is the total number of (active) cells in the model, right?  Not just the number of active cells on the current rank?